src/routes/MapExplorer/MapExplorer.holochain.connector.js

```diff
@@ -0,0 +1,48 @@
+import { connect } from 'react-redux'
```

Comment:
You can get rid of this file and update the `index.js`. I think it'll bomb anyway once you merge in dev if you merged the remove Holochain branch yet.

Reply:
done